taqueria-sdk/index.ts

```diff
@@ -100,6 +101,19 @@ export const getArch = (): LikeAPromise<string, TaqError> => {
 	}
 };
 
+export const getLatestFlextesaImage = async (arch: 'linux/arm64/v8' | 'linux/amd64'): Promise<string> => {
```

Comment:
Beautiful!  Do you mind adding a test for this to the SDK?

Reply:
will do!